portfolio/target/portfolio-1/style.css

```diff
@@ -0,0 +1,4 @@
+#event-end-time-warning {
+  /* default visibility is hidden */ 
```

Comment:
I don't think this comment is necessary.

Reply:
Updated.